src/context/context_mm.h

```diff
@@ -148,11 +148,11 @@ class ContextMemoryManager {
 
 };/* class ContextMemoryManager */
 
-#else /* CVC4_CONTEXT_MEMORY_MANAGER */
+#else /* CVC4_DEBUG_CONTEXT_MEMORY_MANAGER */
 
```

Comment:
(Optional) Add a warning here to alert users that they are using a debug version.

Reply:
Great idea. I added the warning.

Reply:
LGTM